tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "lib": [ "es2021" ],
+    "lib": [ "ES2022" ],
```

Comment:
`ErrorOptions` is an ES2022 feature. However, because the minimum supported version is Node 18, `ES2022` is fine; see https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-18